src/GraphQL/Execution/ExecutionResult.php

```diff
@@ -5,7 +5,8 @@
 use Drupal\Core\Cache\CacheableDependencyInterface;
 use Drupal\Core\Cache\RefinableCacheableDependencyTrait;
 
-class ExecutionResult extends \GraphQL\Executor\ExecutionResult implements CacheableDependencyInterface {
+//@codingStandardsIgnoreStart
+class ExecutionResult extends GraphQL\Executor\ExecutionResult implements CacheableDependencyInterface {
```

Comment:
you removed a backslash here, that's why the tests are failing and PHP cannot find the class,